templates/deno/src/models.d.ts.twig

```diff
@@ -1,5 +1,5 @@
 {% macro sub_schema(property, definition, spec) %}
-{% spaceless %}
+{% apply spaceless %}
```

Comment:
What does these change do? I should probably know so I can work on other sdks 😅

Reply:
the spaceless removes all the white space, previously it was `{% spaceless %}` but I updated the twig to latest version and its now `{% apply spaceless %}`.